lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -285,30 +331,60 @@ sub add_user_ip_geolocation {
     }
 }
 
+$json_schemas{start_domain_test} = joi->object->strict->props(
+    domain => $zm_validator->domain_name->required,
```

Comment:
Could you update the order of the parameters, either here or in the API documentation or both, so that they match up?

Reply:
Done